mimic/test/test_nova.py

```diff
@@ -1374,12 +1374,13 @@ def use_creation_behavior(self, name, parameters, criteria):
         criterion = {"name": name,
                      "parameters": parameters,
                      "criteria": criteria}
-        set_criteria = request(self, self.root, "POST",
-                               self.nova_control_endpoint +
-                               "/behaviors/creation/",
-                               json.dumps(criterion))
-        set_criteria_response = self.successResultOf(set_criteria)
-        self.assertEqual(set_criteria_response.code, 201)
+        set_criteria = json_request(self, self.root, "POST",
+                                    self.nova_control_endpoint +
+                                    "/behaviors/creation/",
+                                    json.dumps(criterion))
+        response, body = self.successResultOf(set_criteria)
+        self.assertEqual(response.code, 201)
+        self.assertTrue(body.get("id"))
```

Comment:
Yep, adding support for deleting behaviors in a future PR.  Would also be good to list at some point.  Will make the test change, thanks!